Parser.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import csv
+import json
+import xml.dom.minidom
+from datetime import datetime
+from datetime import timedelta
+
+
+class Parser:
+    def __init__(self, filename):
+        self.filename = filename
+        self.localTransactions = []
+        self.localAccounts = {}
+
+    def readFile(self):
```

Comment:
could we seperate the readFile out into, readXml, readJson ect. and have the logic for which one we pick in a different function